### DISCUSSION TITLE
It's posible to add Table in a custom view like in the list view?

### DISCUSSION THREAD

**Discussion by jals-es** [filamentphp/filament#4139] [Help] [answered]
Hello, normaly I have in the Resource a table that shows in ListRecord. Like this.

```
public static function table(Table $table): Table
{
        return $table
            ->columns([
                Tables\Columns\TextColumn::make(ModulesInterface::NAME)->searchable(),
                Tables\Columns\BadgeColumn::make(ModulesInterface::AREA)
            ])
}
```

It's posible to add a table like this to a custom view in ViewRecord with a custom data displayed in the table? Thanks.

**Comment by danharrin** [comment, marked as answer]
You can use the Table Builder for this :)

**Reply by SamuelNitsche** [reply]
@danharrin Doesn't look like its possible to populate a table with something else than an eloquent model. At least I couldn't find something in the table builder docs. Is this possible and if so, could you share some simple example? Thanks!

**Reply by danharrin** [reply]
You can use [Sushi](https://github.com/calebporzio/sushi) to transform your data into a model

**Reply by jals-es** [reply]
Yes, I solve it with Table Builder with a Livewire Component, but when I do any action, filter or change pagination, the livewire breaks with an strange error.

I had open this discussion for that.
https://github.com/filamentphp/filament/discussions/4277

**Comment by photojo** [comment]
Would it be possible to get a simple example.
In my "main"-table I use collapsible content (split/panel/stack). I want to show an other table according to the record, to add, edit and remove data.
Is this possible?

**Reply by photojo** [reply]
So, I've implemented a "child"-table via ViewColumn. I can use getTableActions only in the child-table to add, edit, ... records, but I can not do so in the main-table? 
What can I do?
